page-objects/homework.js

```diff
@@ -103,10 +96,8 @@ module.exports = {
     await driver.pause(1000);
   },
   gotoTasks: async function() {
-    let hometasks = await driver.$(
-      'body > aside > nav > ul > li:nth-child(4) > a > i'
-    );
-    await hometasks.click();
+    let url = courseData.urlTasks;
```

Comment:
if you don't assign any other values to a variable, you can use the `const` keyword instead of `let`. This way, you'll be protected against unwanted changes later. (this is true in most places, not only here)